src/wallet/wallet.cpp

```diff
@@ -4072,6 +4072,16 @@ bool CWallet::InitLoadWallet(const std::string& wordlist)
                     return InitError(strprintf(_("You specified a wrong mnemonic")));
                 }
                 masterPubKey = walletInstance->ImportMnemonic(words, lexicon);
+
+                // Check if we gave a password
+                if (!password.empty()) {
+                    // Is this safe? or do I need to do something more
+                    // to the string after I've converted it to SecureString
+                    SecureString strWalletPass;
+                    strWalletPass.reserve(100);
+                    strWalletPass = password.c_str();
+                    walletInstance->EncryptWallet(strWalletPass);
+                }
```

Comment:
@aguycalled About this part, is this safe? will it not lead the password in memory?